server_installation/topics/operating-mode/crossdc/proc-setting-up-infinispan.adoc

```diff
@@ -0,0 +1,172 @@
+----
+<cache-container name="default" statistics="true">
+      <transport cluster="${infinispan.cluster.name:cluster}"
+                 stack="xsite"/> <1>
+</cache-container>
+----
+<1> Uses the `xsite` stack for the cluster.
++
+. Configure the keystore as an SSL identity in the server security realm.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<server-identities>
+  <ssl>
+    <keystore path="server.jks" <1>
+              relative-to="infinispan.server.config.path"
+              keystore-password="password" <2>
+              alias="server" /> <3>
+  </ssl>
+</server-identities>
+----
+<1> Specifies the path of the keystore that contains the SSL identity.
+<2> Specifies the password to access the keystore.
+<3> Names the alias of the certificate in the keystore.
++
+. Configure the authentication mechanism for the Hot Rod endpoint.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<endpoints socket-binding="default" security-realm="default">
+   <hotrod-connector name="hotrod">
+      <authentication>
+         <sasl mechanisms="SCRAM-SHA-512" <1>
+               server-name="infinispan" /> <2>
+      </authentication>
+   </hotrod-connector>
+   <rest-connector name="rest"/>
+</endpoints>
+----
+<1> Configures the SASL authentication mechanism for the Hot Rod endpoint.  SCRAM-SHA-512 is the default SASL mechanism for Hot Rod. However you can use whatever is appropriate for your environment, such as GSSAPI.
+<2> Defines the name that {jdgserver_name} servers present to clients. You specify this name in the Hot Rod client configuration when you set up {project_name}.
++
+. Create a cache template.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<cache-container ... >
+  <replicated-cache-configuration name="sessions-cfg" <1>
+                                  mode="SYNC"> <2>
+    <locking acquire-timeout="0" /> <3>
+    <backups>
+      <backup site="site2" strategy="ASYNC" /> <4>
+    </backups>
+  </replicated-cache-configuration>
+</cache-container>
+----
+<1> Creates a cache template named `sessions-cfg`.
+<2> Defines a cache that synchronously replicates data across the cluster.
+<3> Disables timeout for lock acquisition.
+<4> Names the backup site for the {jdgserver_name} cluster you are configuring.
++
+. Repeat the preceding steps to modify `infinispan.xml` on each node in the {jdgserver_name} cluster.
+. Start the cluster and open {jdgserver_name} Console in any browser.
+. Authenticate with the {jdgserver_name} user you created.
+. Add caches for {jdgserver_name} data using the `sessions-cfg` cache template.
+.. Navigate to the *Data Container* tab and then select *Create Cache*.
+.. Specify a name for the cache.
+.. Select `sessions-cfg` from the *Template* drop-down menu.
+.. Select *Create*.
++
+Use preceding steps to create each of the following caches:
++
+* work
+* sessions
+* clientSessions
+* offlineSessions
+* offlineClientSessions
+* actionTokens
+* loginFailures
+
```

Comment:
Missing step for how to start Infinispan servers and verifying their cross-site setup configuration.

Reply:
Could be more than one step.  @tkyjovsk can you or  @oraNod provide the official instructions.

Reply:
@andymunro Here are instructions for starting DG server instances: https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.1/html-single/data_grid_server_guide/index#starting_data_grid_servers

This topic explains how to verify xsite views: https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.1/html-single/data_grid_guide_to_cross-site_replication/index#verify_xsite_view-xsite